test/mock-kubectl

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+[[ -n $DEBUG ]] && set -x
+
+set -eou pipefail
+
+if [[ $@ == *'get namespaces'* ]]; then
+  echo "default"
```

Comment:
let's call these ns1 and ns2. no need to stick to original names (hard to follow in test cases above), and no need to have 3.